Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1083,9 +1085,11 @@ public function toJson()
      */
     public function __clone()
     {
-        foreach ($this as $key => $val) {
-            $this[$key] = clone $val;
+        foreach ($this as $key => &$val) {
+            $val = clone $val;
         }
+        // unset reference to keep scope clear
+        unset($val);
```

Comment:
Uhh, good it was not in release.

Where did this happen - I want to look into details, probably that due to clone, but also probably new issue I have learn from?